src/pytest_benchmark/elasticsearch_storage.py

```diff
@@ -11,7 +32,7 @@ def __init__(self, elasticsearch_hosts, elasticsearch_index, elasticsearch_docty
         self._elasticsearch_hosts = elasticsearch_hosts
         self._elasticsearch_index = elasticsearch_index
         self._elasticsearch_doctype = elasticsearch_doctype
-        self._elasticsearch = elasticsearch.Elasticsearch(self._elasticsearch_hosts)
+        self._elasticsearch = elasticsearch.Elasticsearch(self._elasticsearch_hosts, serializer=SaveElasticsearchJSONSerializer())
```

Comment:
Note that this will create an error at runtime if elasticsearch could not be imported (`None` is not callable).